### PR TITLE
Data protection reference: Document use of data-at-rest encryption

### PR DESCRIPTION
**Pull request by osnyx**
This updates our public documentation to reflect the successful rollout of PL-131325 with the deploayment of disk encryption in our storage.

Changes to out data protection reference require special caution and need to be reviewed by @zagy or @ctheune. Until then, keeping this as draft.

### CODE REVIEW

File: src/reference/security/data-protection.md [hunk 1]
```diff
@@ -197,6 +197,16 @@ operations to ensure traceability of actions.
 Passwords for physical machines granting access to root accounts and IPMI
 controllers are stored as copies in a strongly encrypted password manager.
 
+(data-at-rest-encryption)=
+
+### Data-at-Rest Encryption
+
+All data in our [storage cluster](#infrastructure-storage) is stored encrypted. This includes the *Virtual Disk Block Storage*, our *S3-compatible Object Storage*, and [all automated backups](#backup). Whenever storage media are located outside our trusted areas, we have technical and organisational measures in place to ensure the confidentiality of the data stored there.
+
+On a technical level, we utilise the common Linux [cryptsetup](https://gitlab.com/cryptsetup/cryptsetup) software stack with LUKS2 metadata and XTS-AES-512 cipher. We regularly review recommendations in standard guidelines or working groups and are able to update the used encryption parameters once necessary.
```
Comment (osnyx) [inline src/reference/security/data-protection.md:206]: We could get into more detail on which recommendations (e.g. BSI, ENISA) we follow, we have to decide on whether this is the appropriate place to do so or not.

Comment (osnyx) [reply src/reference/security/data-protection.md:206]:
```suggestion
On a technical level, we utilise the common Linux [cryptsetup](https://gitlab.com/cryptsetup/cryptsetup) software stack with LUKS2 metadata and XTS-AES-512 cipher. We regularly review recommendations in standard guidelines or by working groups and are able to update the used encryption parameters once necessary.
```

Comment (ctheune) [reply src/reference/security/data-protection.md:206]: not needed.

**Review by ctheune — APPROVED** [review]
LGTM, my changes have been incorporated

**Review by zagy — APPROVED** [review]

**Pull request merged**